redis/Makefile

```diff
@@ -0,0 +1,98 @@
+
+REDIS_VERSION:=3.0.7
```

Comment:
It's taken from my code which is MIT licensed. Do I need to add the OGM license header to this file?